gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,82 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data, :standby_host
+
+    def initialize(hash = {})
+      self.appliance_pg_data = ENV.fetch('APPLIANCE_PG_DATA')
+      set_defaults
+      super
+    end
+
+    def set_defaults
+      self.standby_host = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+      self.host         = nil
+      self.database     = "vmdb_production"
+      self.username     = "root"
+      self.password     = nil
+      self.cluster_name = "replication_cluster"
+      self.node_number  = 1
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Replication Standby Server\n")
+      ask_for_standby_host
+      ask_for_cluster_name
+      ask_for_node_number
+      ask_for_database_credentials
+      confirm
+    end
+
+    def ask_for_standby_host
+      ask_for_ip_or_hostname("Standby Server hostname or IP address", standby_host) if standby_host.blank? || !local?
+    end
+
+    def activate
+      say("Configuring Replication Standby Server...")
+      confirm_no_database? &&
+        already_configured? &&
+        create_config_file(standby_host) &&
+        clone_standby_server &&
+        start_postgres &&
+        register_standby_server
+    end
+
+    def confirm_no_database?
+      return true if Dir.glob("#{appliance_pg_data}/*").empty?
+      say ""
+      say("Appliance database found under: #{appliance_pg_data}")
+      say("Replication standby server can not be configured if the database already exists")
+      say("Remove the existing database before configuring as a standby server")
+      false
+    end
+
+    def clone_standby_server
+      params = { :h  => host,
```

Comment:
@carbonin It's actually correct but admittedly confusing. I'll rework the host nameing.